app/__init__.py

```diff
@@ -1,22 +1,25 @@
 from flask import Flask
 from .db import db, migrate
 from .models import task, goal
+from .routes.task_routes import bp as tasks_bp
+from .routes.goal_routes import bp as goals_bp
 import os
+from dotenv import load_dotenv
 
 def create_app(config=None):
+    load_dotenv()
```

Comment:
```suggestion

def create_app(config=None):
```

We don't need `load_dotenv()` to get the environment variables when we start our Flask app with `flask run`. We only need to explicitly load our values from .env and have them be environment variables when we run our tests with `pytest`. That's why we only need to have `load_dotenv()` in `conftest.py`